pkg/login/social/social.go

```diff
@@ -206,6 +212,7 @@ type BasicUserInfo struct {
 	Company string
 	Role    string
 	Groups  []string
+	Teams   []string
```

Comment:
Should this be

```go
Teams []int
```

since that's what is expected in the JSON?

Reply:
No, an array of strings is expected. Not sure where you get the int from. It used to be ints but that is insufficiently generic.